README.md

```diff
@@ -222,33 +222,49 @@ Alternative syntaxes for those two cases each have their own flaws; and deciding
 
 As for &lt;language X>, it has different syntactical constraints than JavaScript because of &lt;some construct not supported by X or working differently in X>.
 
-<dt>Ok, but I <b>really</b> think that <i>&lt;alternative syntax></i> is better
+</dd>
+
+<dt>Ok, but I really think that &lt;alternative syntax> is better.</dt>
   
 <dd>
   
 Various alternative syntaxes has been explored and extensively discussed in the past. None of them gained consensus. Search for [issues
 with label “alternative syntax”](https://github.com/tc39/proposal-optional-chaining/issues?utf8=%E2%9C%93&q=label%3A%22alternative+syntax%22), as well as [issues
 with label “alternative syntax and semantics”](https://github.com/tc39/proposal-optional-chaining/issues?utf8=%E2%9C%93&q=label%3A%22alternative+syntax+and+semantics%22) for those that had impact on semantics.
 
-<dt>Why does (null)?.b evaluate to undefined rather than null?
+</dd>
+
+<dt>Why does <code>(null)?.b</code> evaluate to <code>undefined</code> rather than <code>null</code>?</dt>
 
 <dd>
 
 Neither `a.b` nor `a?.b` is intended to preserve arbitrary information on the base object `a`, but only to give information about the property `"b"` of that object. If a property `"b"` is absent from `a`, this is reflected by `a.b === undefined` and `a?.b === undefined`.
 
 In particular, the value `null` is considered to have no properties; therefore, `(null)?.b` is undefined.
 
+</dd>
+
+<dt>Why does <code>foo?.()</code> throw when foo is neither nullish nor callable?</dt>
+
+<dd>
+
+Imagine a library which will call a handler function, e.g. `onChange`, just when the user has provided it. If the user provides the number `3` instead of a function, the library will likely want to throw and inform the user of their mistaken usage. This is exactly what the proposed semantics for `onChange?.()` achieve.
 
+Moreover, this ensures that `?.` has a consistent meaning in all cases. Instead of making calls a special case where we check `typeof foo === 'function'`, we simply check `foo == null` across the board.
```

Comment:
I might combine this and the next sentence, into something like:

> furthermore, this ensures that `?.` has consistent behavior with other cases like property access, i.e., if `this.onChange` is not callable, `this.onChange()` will throw a TypeError; this is consistent with the fact that optional chaining is [not an error-suppression mechanism](#is-this-error-suppression).

Take or leave my suggestion 😄

Reply:
I really do want the consistency argument to stand on its own though—the fact that `a?.b`, `a?.[b]`, and `a?.(b)` all perform the same check is an important point separate from error concerns. :smile:

Reply:
I'm sold 👍... my only concern was that people take the decision to be driven by simplicity to implement, vs., elegance; this approach feels both the simplest to implement and most elegant to me, and I'm probably overthinking.

Reply:
@claudepache Would you rather that we put the two paragraphs back in their original order then (sans the word "primary")? I think that's the last remaining concern here.

Reply:
@bcoe Note that simplicity to implement has never been a factor in this discussion. All the alternatives we discussed for this proposal should be relatively simple to implement. The tradeoffs are all at the design level.